Makefile

```diff
@@ -241,7 +242,13 @@ buildEcsDeployer:
 
 ## Deploy all infrastructure
 deploy: dockerCheck dockerLogin cleanMisc modelCheck buildEcsDeployer
-ifdef PROFILE
+ifneq (,$(findstring true, $(HEADLESS)))
+	if [ -z ${PROFILE} ]; then \
+        npx cdk deploy ${STACK} -c ${ENV}='$(shell echo '${${ENV}}')'; \
+    else \
+        npx cdk deploy ${STACK} --profile ${PROFILE} -c ${ENV}='$(shell echo '${${ENV}}')'; \
```

Comment:
We will want `cdk deploy --require-approval never` added so the deployer doesn't have to confirm stack changes.